src/poke_env/data/static/moves/gen1moves.json

```diff
@@ -573,15 +579,15 @@
         "num": 68,
         "onTry": "onTry",
         "pp": 20,
-        "priority": -1,
+        "priority": -5,
         "secondary": null,
         "target": "scripted",
         "type": "Fighting",
         "willCrit": false
     },
     "crabhammer": {
         "accuracy": 85,
-        "basePower": 90,
+        "basePower": 100,
```

Comment:
How did that happen? I think this is incorrect - crabhammer had a base power of 90 in gen1